CODEOWNERS

```diff
@@ -10,8 +10,12 @@
 # All developers working on this repository are able to edit main values.yaml file.
 /chart/compass/values.yaml @PK85 @aszecowka @crabtree @pkosiec @kfurgol @tgorgol @akgalwas @janmedrek @Szymongib @franpog859 @Maladie @dbadura @kjaksik @mszostok @piotrmiskiewicz @polskikiel @ksputo @jasiu001 @koala7659
 
+# Registration job is used by provisioner and kyma-environment-broker
+/chart/compass/templates/registration-job.yaml @PK85 @aszecowka @crabtree @pkosiec @kfurgol @tgorgol @akgalwas @janmedrek @Szymongib @franpog859 @Maladie @dbadura @kjaksik @mszostok @piotrmiskiewicz @polskikiel @ksputo @jasiu001 @koala7659
+
 # All .md files
 *.md @klaudiagrz @kazydek @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova
+/docs/kyma-environment-broker @klaudiagrz @kazydek @tomekpapiernik @bszwarc @mmitoraj @majakurcius @alexandra-simeonova @PK85 @mszostok @piotrmiskiewicz @polskikiel @ksputo @jasiu001
```

Comment:
AFAIK you don't have to specify Technical Writers again - specify just your team, move this line above the `*.md` line and everything should work fine 🙂

Reply:
https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax

Reply:
ok, I changed that but I moved this line below, under the # kyma-environment-broker comment. I think it suits there. WDYT?